apps/explorer/src/components/Activity/components/TransactionsActivityTable.tsx

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
I don't love nested components/ folders, so lets pull this into the parent folder